pilot/pkg/model/context.go

```diff
@@ -606,6 +606,10 @@ type NodeMetadata struct {
 	Raw map[string]interface{} `json:"-"`
 }
 
+func (m *NodeMetadata) HasConfiguredCerts() bool {
```

Comment:
Comment ? Can you move this to cluster.go and keep it private, it is only for backward compat and I would not pollute 'model' with things depending on node metadata.

I don't think we have a clear API for 'configured certs' or I know what it really means.

Reply:
I can move that out cluster